model/README.md

```diff
@@ -0,0 +1,7 @@
+# Model
+
+Package `model` provides modeling components, such as domains and ranges.
+
+See [godocs][godocs] for package docs.
+
+[godocs]: ./doc.go
```

Comment:
```suggestion
[godocs]: https://pkg.go.dev/github.com/nextmv-io/sdk/model
```
Will suggest this in the other files, but only make an actual comment here. If someone was browsing around in the repo and stumbles upon this link, in my mind it is better if they go to the "pretty" rendered site rather than staring at some literal comments.